delivery_label_direct_print/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2014 Camptocamp SA
```

Comment:
Yes of course.
But actually, the original module has just been migrated as this so I didn't change dates.
Should I do it ?